### PR TITLE
Deploy cert-checker to monitoring cluster to check all certs keytabs

### PR DESCRIPTION
**Pull request by mrceyhun**
Cert-checker implementation to check all monitoring certificates and keytabs. Deployed in cmsmonitnew-cluster-vef5nqhek4nf-node-0:30901 . Metrics: `cert_valid_sec` , `keytab_valid_sec`.

Used repo: https://github.com/vkuznet/cert-checker with some modifications in https://github.com/vkuznet/cert-checker/pull/1.

### CODE REVIEW

File: kubernetes/monitoring/services/cert-checker.yaml
Comment (vkuznet) [inline kubernetes/monitoring/services/cert-checker.yaml:57]: please do not use concrete image tag and rather replace `:test` with ` #imagetag` which we use in `deploy-srv.sh` script when we deploy services

**Review by vkuznet — CHANGES_REQUESTED** [review]
in addition, I ask @muhammadimranfarooqi to review these changes too

**Comment by mrceyhun** [comment]
@vkuznet I applied the changes you suggested in https://github.com/vkuznet/cert-checker/pull/1 and create a new image with test tag again. It is deployed now. After my PR merged into your repo, we can continue to use `cmsweb/cert-checker` with new build. Then I can remove `cmsmonitoring/cert-checker` one which is just for test.

You can see that empty string keytab and certs are removed from the configs.json. Waiting for your green light and new `cmsweb/cert-checker` build with my PR changes. Then it will be deployed to production.

CMSWEB probably will use extra parameters like `#imagetag`, `env` label in service definition as you said. One thing is that, each request checks all the certs and keytabs , therefore we should not set livenessProbe period as 3 seconds. I set 600s, but above 30s is fine IMO.

**Comment by vkuznet** [comment]
@mrceyhun please feel free to proceed, I merged your PR.

**Comment by vkuznet** [comment]
@muhammadimranfarooqi , please review and merge this PR.

**Pull request merged**
